clang/include/clang/Format/Format.h

```diff
@@ -4648,6 +4648,10 @@ struct FormatStyle {
   /// \version 11
   std::vector<std::string> WhitespaceSensitiveMacros;
 
+  /// Ignore formatting in preprocessor definitions.
+  /// \version 18
+  bool IgnorePPDefinitions;
```

Comment:
Please sort alphabetically.

Reply:
(this applies to not just here, but below on line 4818, and in Format.cpp, and in ConfigParseTest.cpp)

Reply:
fixed